tests/model_registry/rbac/utils.py

```diff
@@ -178,3 +181,16 @@ def assert_forbidden_access(endpoint: str, token: str) -> None:
     except ForbiddenException:
         # This is what we want - access is properly forbidden
         pass
+
+
+def should_skip_rbac_tests() -> bool:
+    """Check if RBAC tests should be skipped"""
+    # Need to get these details here to use this in the skipif statement.
+    # Skip RBAC tests on OpenShift 4.20+ with OIDC authentication
+    success, auth_type, _ = run_command(command=["oc", "get", "authentication", "-o", "jsonpath={.items[0].spec.type}"])
+    if not success:
+        LOGGER.warning("Failed to retrieve authentication type, assuming non-OIDC")
+        return False
+    if auth_type and auth_type.strip() != "OIDC":
+        return False
+    return get_cluster_version() >= Version.parse("4.20.0")
```

Comment:
Please use wrapper authentication object. Please look at https://github.com/opendatahub-io/opendatahub-tests/blob/main/tests/conftest.py#L380. Wrapper changes are merged and we updated the uv.lock file. So you should be able to use it directly.